packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -89,6 +90,22 @@ export type TablePaginatorRendererProps = {
   isLoading?: boolean;
 };
 
+type ScrollToItemProps<T extends Record<string, unknown>> =
+  | {
+      /**
+       * Initialize the table scrolled to this item.
+       *
+       * Must be used in conjunction with `getRowId` prop.
+       *
+       * Does not support tables with pagination.
+       */
+      scrollToItem: T;
```

Comment:
Shouldn't it accept a string instead which is row id?

Reply:
It could work either way. I figured it would be easier for the consumer to pass the item. Possibly more performant if we pass the row ID, a we don't need to run `getRowId` on every item until we find it, but that may be premature optimization.
https://github.com/iTwin/iTwinUI-react/pull/689#discussion_r891675269